libs/application-generic/src/commands/base.command.ts

```diff
@@ -15,7 +15,7 @@ export abstract class BaseCommand {
       throw new CommandValidationException(this.name, flattenedErrors);
     }
 
-    return data;
+    return convertedObject;
```

Comment:
it is needed in order to get the default in the class like we have here 
```
  allowUpdate?: boolean = true;
 ```
 otherwise the value is undefined.
 any objection to this change?

Reply:
That is the correct value we should have here 🤔 @tatarco it seem to be modified here: https://github.com/novuhq/novu/pull/7173 is there any reason for this?